aries_cloudagent/protocols/present_proof/v1_0/handlers/presentation_request_handler.py

```diff
@@ -34,9 +33,6 @@ class PresentationRequestHandler(BaseHandler):
             context.message.serialize(as_string=True)
         )
 
```

Comment:
I'm not sure why the connection-ready check is being removed for this handler? There's also references to the connection record later which look like they would fail if there wasn't one identified.

Reply:
You are right about the references further down the code, good catch. It is very odd that an exception is not being thrown...

I am pretty sure the presentation request was failing because of this connection readiness check, and that is why I had to remove it. We could re-test the flow after reverting the change to double check if it ireally is necessary.

Reply:
You were correct. I re-tested everything after reverting this change and it seems to work just fine - not sure why the error was coming up in the logs (maybe I got confused between this and the presentation_handler).